charts/mc-router/templates/deployment.yaml

```diff
@@ -0,0 +1,151 @@
+          env:
+            {{- include "mc-router.envBoolMap" (list "IN_KUBE_CLUSTER" "true") | nindent 12 }}
+            {{- include "mc-router.envMap" (list "API_BINDING" (printf ":%d" (.Values.service.apiPort | int))) | nindent 12 }}
+            {{- include "mc-router.envMap" (list "PORT" .Values.service.minecraftPort) | nindent 12 }}
+
+            {{- with .Values.minecraftRouter }}
+              {{- include "mc-router.envBoolMap" (list "AUTO_SCALE_UP" .autoScaleUp.enabled) | nindent 12 }}
+              {{- include "mc-router.envMap" (list "CONNECTION_RATE_LIMIT" .connectionRateLimit) | nindent 12 }}
+              {{- include "mc-router.envMap" (list "CPU_PROFILE" .cpuProfilePath) | nindent 12 }}
+              {{- include "mc-router.envBoolMap" (list "DEBUG" .debug.enabled) | nindent 12 }}
+
+              {{- with .defaultServer }}
+                {{- include "mc-router.envMap" (list "DEFAULT" (printf "%s:%d" .host (.port | int))) | nindent 12 }}
+              {{- end }}
+
+              {{- include "mc-router.envMultilineMap" (list "MAPPING" (include "mc-router.formatMappings" .mappings)) | nindent 12 }}
+
+              {{- with .metrics }}
+                {{- include "mc-router.envMap" (list "METRICS_BACKEND" .backend) | nindent 12 }}
+
+                {{- if eq .backend "influxdb" }}
+                  {{- with .influxdb }}
+                    {{- include "mc-router.envMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_ADDR" .address) | nindent 12 }}
+                    {{- include "mc-router.envMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_DATABASE" .database) | nindent 12 }}
+                    {{- include "mc-router.envMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_INTERVAL" .interval) | nindent 12 }}
+                    {{- include "mc-router.envMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_RETENTION_POLICY" .retentionPolicy) | nindent 12 }}
+                    {{- include "mc-router.envMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_TAGS" .tags) | nindent 12 }}
+                    {{- with .credentials }}
+                      {{- include "mc-router.envSecretMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_USERNAME" .existingSecret .usernameKey) | nindent 12 }}
+                      {{- include "mc-router.envSecretMap" (list "METRICS_BACKEND_CONFIG_INFLUXDB_PASSWORD" .existingSecret .passwordKey) | nindent 12 }}
+                    {{- end }}
+                  {{- end }}
+                {{- end }}
+              {{- end }}
+
+              {{- include "mc-router.envSecretMap" (list "NGROK_TOKEN" .ngrokToken.existingSecret .ngrokToken.tokenKey ) | nindent 12 }}
+              {{- include "mc-router.envBoolMap" (list "SIMPLIFY_SRV" .simplifySrv) | nindent 12 }}
+              {{- include "mc-router.envBoolMap" (list "USE_PROXY_PROTOCOL" .useProxyProtocol) | nindent 12 }}
+              {{- include "mc-router.envBoolMap" (list "VERSION" .showVersion) | nindent 12 }}
+            {{- end }}
```

Comment:
Can you include an "extra env" section like

https://github.com/itzg/minecraft-server-charts/blob/1e56e34e883b53b2eac145c75df4971d4c2d71a0/charts/minecraft/templates/deployment.yaml#L363-L374

which serves as a nice escape hatch when new env vars are introduced before the chart is updated, if at all.

Reply:
Done!